src/test/java/sorinfoca/Tests/Tema7/TestConfig.java

```diff
@@ -0,0 +1,46 @@
+package sorinfoca.Tests.Tema7;
+
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.*;
+
+public class TestConfig {
+
+    protected ChromeDriver driver = null;
+    private static String baseUrl = "https://katalon-demo-cura.herokuapp.com/";
+
+    public static String getBaseUrl() {
+        return baseUrl;
+    }
+
+    @BeforeTest
+    public void beforeTest() { System.out.println("Before test"); }
+
+    @AfterTest
+    public void afterTest() { System.out.println("After test"); }
+
+    @BeforeClass
+    public void beforeClass() {
+        System.out.println("Before class");
+    }
+
+    @BeforeSuite
+    public void beforeSuite() {
+        System.out.println("Before suite");
+    }
+
+    @AfterSuite
+    public void afterSuite() {
+        System.out.println("After suite");
+    }
+
+    @BeforeMethod
+    public void setup() {
+        System.out.println("Before method");
```

Comment:
Can you try to add in this method the step for opening the browser in order to remove it from each of the test cases methods? We avoid in this way duplicating the line `driver = BrowserManager.createChromeDriver();` in MyTests.java class. You have an example in demo videos for chapter 7 how you can do that. If you have any issues, let me know. Thanks!